tools/ci_build/github/linux/docker/scripts/install_os_deps.sh

```diff
@@ -19,7 +19,11 @@ done
 
 DEVICE_TYPE=${DEVICE_TYPE:=Normal}
 
-#Download a file from internet
+# Development tools and libraries
+apt install -y \
```

Comment:
You may move it to install_ubuntu.sh

Reply:
This file is shared by centos and ubuntu. But I feel centos images should use the one in the manylinux folder.

Reply:
just removed it